src/routes/home/components/__tests__/DisciplineCard.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { mocked } from 'jest-mock'
+import React from 'react'
+
+import labels from '../../../../constants/labels.json'
+import useReadProgress from '../../../../hooks/useReadProgress'
+import { getReturnOf } from '../../../../testing/helper'
+import { mockDisciplines } from '../../../../testing/mockDiscipline'
+import render from '../../../../testing/render'
+import DisciplineCard from '../DisciplineCard'
+
+jest.mock('../../../../hooks/useReadProgress')
+
+describe('DisciplineCard', () => {
+  const navigate = jest.fn()
+  it('should show progress if enabled', async () => {
+    mocked(useReadProgress).mockReturnValueOnce(getReturnOf(1))
+    const { findByText, findByTestId } = render(
+      <DisciplineCard discipline={mockDisciplines[0]} showProgress navigateToNextExercise={navigate} />
+    )
+    expect(await findByText(mockDisciplines[0].title)).toBeDefined()
```

Comment:
same as above, you can use `getByText` `getByTestId` without await and async fkt